submissions/mxmgny/a-tiny-js-world/index.js

```diff
@@ -0,0 +1,43 @@
+let cat = {
+    appearance: "cat",
+    name: "Garfield",
+    legs: 4,
+    hands: 0,
+    gender: "male",
+    shout: "I love lazagna!",
+    friends:['Selina', 'Jeremy']
+};
+let cat_woman = {
+    appearance:"cat-woman",
+    name:"Selina",
+    legs:2,
+    hands:2,
+    gender: "female",
+    shout:cat['shout'],
+    friends: [cat['name'],'Sansa']
```

Comment:
Use dot notation to access object properties. Each time - your code goes three characters shorter. Also, it is more readable way (`cat.name` instead of `cat['name']`). Meet the [article](https://codeburst.io/javascript-quickie-dot-notation-vs-bracket-notation-333641c0f781) to know, which cases you **must** use bracket notation.

Reply:
Done